pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDaoImpl.java

```diff
@@ -515,6 +522,30 @@ private boolean booleanCheckFoundAsName(String kitLabel, String query) {
         return (int) results.resultValue > 0;
     }
 
-
-
+    @Override
+    public Optional<ScanError> updateKitByHruid(KitRequestShipping kitRequestShipping) {
+        Optional<ScanError> result = Optional.empty();
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult dbVals = new SimpleResult();
+            try (PreparedStatement stmt = conn.prepareStatement(UPDATE_KIT_BY_HRUID.replace("#", kitRequestShipping.getShortId()))) {
```

Comment:
replacing `#` may become victim of sql injection, is it not possible to do it with `?`?